src/backend/distributed/citus--6.1-17--6.2-1.sql

```diff
@@ -0,0 +1,26 @@
+/* citus--6.1-17--6.2-1.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION citus_table_size(logicalrelid regclass)
+    RETURNS bigint
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_table_size$$;
+COMMENT ON FUNCTION citus_table_size(logicalrelid regclass)
+    IS 'get the size of distributed tables';    
+    
+CREATE FUNCTION citus_relation_size(logicalrelid regclass)
+    RETURNS bigint
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_relation_size$$;
+COMMENT ON FUNCTION citus_relation_size(logicalrelid regclass)
+    IS 'get a relation''s ''main'' fork''s size';
+    
+CREATE FUNCTION citus_total_relation_size(logicalrelid regclass)
+    RETURNS bigint
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$citus_total_relation_size$$;
+COMMENT ON FUNCTION citus_total_relation_size(logicalrelid regclass)
+    IS 'get distributed table and its indexes'' total relation size';
+    
+RESET search_path;
```

Comment:
Could you add an empty line here and then delete it with backspace. :smile: That stop sign means that there is not an new line character(different than empty line) at the end of the file.